### PR TITLE
add intersect method

### PR DESCRIPTION
**Pull request by thekuom**
knex PR: https://github.com/tgriesser/knex/pull/3023

### CODE REVIEW

File: sections/builder.js [hunk 1]
```diff
@@ -1356,6 +1356,39 @@ export default [
       }
     ]
   },
+  {
+    type: "method",
+    method: "intersect",
+    example: ".intersect([*queries], [wrap])",
```
Comment (kibertoad) [inline sections/builder.js]: what does the * in *queries stand for?

Comment (thekuom) [reply sections/builder.js]: I believe that is denoting that we can call intersect like
```
knex(tableName).intersect(callback1, callback2, callback3)
```

There is similar syntax using the `*` in `select`, `union`, `dropColumns`, and `groupBy`

Comment (kibertoad) [reply sections/builder.js]: Ah, I see, this means "variable amount of".

**Comment by thekuom** [comment]
Should we revisit this since `intersect()` has been released?

**Pull request merged**